files/install-deps-worker.yaml

```diff
@@ -39,6 +39,7 @@
           - fedpkg-stage
           - postgresql # pg_dump
           - python3-boto3 # AWS (S3)
+          - python3-fasjson-client
```

Comment:
Do we still need the `python3-fedora`? The comment (20 lines above) says "to access FAS".

Reply:
you are right, I think we shouldn't need it anymore

Reply:
OTOH, we've probably depended on it to pull in also `fedpkg` and `python3-bodhi-client` as I don't see those in the list. But it's not an excuse to have it there, because it pulls in other dependencies as well.

Reply:
hmm, so you propose to replace `python3-fedora` with `fedpkg` and `python3-bodhi-client`? or do I get it wrong? :D

Reply:
yes, exactly, and then `make build-test-image && make check-in-container` to see if we need something else.

Reply:
I just did `make build-test-image && make check-in-container` only with removing `python3-fedora` (and not adding the 2 packages) and it is weird that the tests passed even without those :thinking: